testing/test_config.py

```diff
@@ -595,14 +595,14 @@ def test_getoption(self, pytester: Pytester) -> None:
     def test_getconftest_pathlist(self, pytester: Pytester, tmp_path: Path) -> None:
         somepath = tmp_path.joinpath("x", "y", "z")
         p = tmp_path.joinpath("conftest.py")
-        p.write_text(f"pathlist = ['.', {str(somepath)!r}]")
+        p.write_text(f"mylist = ['.', {str(somepath)!r}]")
```

Comment:
how about we use `str` of `list` for clarity

```suggestion
        p.write_text(f"mylist = {['.', os.fs_path(somepath)]}")
```

Reply:
Updated, thanks